server/jetstream_api.go

```diff
@@ -480,7 +480,13 @@ func (s *Server) jsTemplateCreateRequest(sub *subscription, c *client, subject,
 		return
 	}
 
-	var resp = JSApiStreamTemplateCreateResponse{ApiResponse: ApiResponse{Type: JSApiStreamTemplateCreateResponseType}}
+	var resp = JSApiStreamTemplateCreateResponse{
+		ApiResponse: ApiResponse{Type: JSApiStreamTemplateCreateResponseType},
+		StreamTemplateInfo: &StreamTemplateInfo{
+			Streams: []string{},
```

Comment:
when go json marshal a unset []string it puts
null in the body and not [] which then makes
it an invalid list of strings.